lib/@empirica/core/src/admin/classic/proxy.ts

```diff
@@ -111,4 +111,41 @@ export class ClassicListenersCollector extends ListenersCollector<
       }
     );
   }
+
+  onPlayerConnected(cb: (props: { player: Player }) => void) {
+
+    console.log('onPlayerConnected/call', cb);
+    
+    this.on(
+      TajribaEvent.ParticipantConnect,
+    (
+        _: EventContext<Context, ClassicKinds>,
+        // { player }: { player: Player }
+        context
+      ) => {
+          console.log('_', _);
+          
+          cb(context);
+        }
+      );
```

Comment:
I'm not sure why this participant connection callback is created inside the player connection callback.